dockerfiles/startall.sh

```diff
@@ -95,10 +105,11 @@ if [[ -n ${PERSIST} ]] ; then
   mkdir -p ${PERSIST_PATH}/hpcc-data
```

Comment:
minor: could also execute this if -n ${PERSISTVALUES}

Reply:
Should really be creating directories that are named in the values file in that case